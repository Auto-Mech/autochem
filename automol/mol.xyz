 10

C    2.991675   0.448392   0.125994
C   -2.991674  -0.448395   0.228526
C    1.857709   0.054944   0.111820
C   -1.857710  -0.054942   0.242694
C    0.517575  -0.416325   0.094126
C   -0.517576   0.416329   0.260384
H    3.998048   0.797314   0.138536
H   -3.998045  -0.797323   0.215992
H    0.357980  -1.479784  -0.059440
H   -0.357982   1.479788   0.413949

