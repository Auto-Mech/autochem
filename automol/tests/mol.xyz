14

C       -1.8253241225     -1.5444597008      0.0030629375                 
C       -0.2995009688     -1.5317757153      0.0028893532                 
H       -2.1889216391     -2.5935419033      0.0173206739                 
H       -2.2157238223     -1.0451212981     -0.9092781509                 
H       -2.2157224169     -1.0205121040      0.9014977498                 
H        0.0637680645     -2.0805067098     -0.8934255005                 
H        0.0637695242     -2.0559460278      0.9137847632                 
C        0.2413728602     -0.0978420118     -0.0165987838                 
H       -0.1218977093      0.4263283435     -0.9274941367                 
C        1.7671960140     -0.0851580313     -0.0167725543                 
H       -0.1218960956      0.4508889423      0.8797161281                 
H        2.1575941729     -0.6091055354     -0.9152074814                 
H        2.1575958470     -0.5844965320      0.8955684217                 
H        2.1307935333      0.9639241708     -0.0310302486                 
