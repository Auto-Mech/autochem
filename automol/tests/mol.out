molecule is nonlinear
has enantiomer? no

rotational symmetry number = 2

Molecular structure:

   A\A    C1   C2   H3   H4   H5   H6   H7   C8   H9   C10  H11  H12  H13  H14

   C1     X

   C2     1    X

   H3     1    0    X

   H4     1    0    0    X

   H5     1    0    0    0    X

   H6     0    1    0    0    0    X

   H7     0    1    0    0    0    0    X

   C8     0    1    0    0    0    0    0    X

   H9     0    0    0    0    0    0    0    1    X

   C10    0    0    0    0    0    0    0    1    0    X

   H11    0    0    0    0    0    0    0    1    0    0    X

   H12    0    0    0    0    0    0    0    0    0    1    0    X

   H13    0    0    0    0    0    0    0    0    0    1    0    0    X

   H14    0    0    0    0    0    0    0    0    0    1    0    0    0    X




Z-Matrix atom order:
 1 -->  1
 2 -->  2
 3 -->  3
 4 -->  4
 5 -->  5
 6 -->  6
 7 -->  7
 8 -->  8
 9 -->  9
10 --> 10
11 --> 11
12 --> 12
13 --> 13
14 --> 14

Z-Matrix:
C 
C , 1 , R1 
H , 1 , R2 , 2 , A2 
H , 1 , R3 , 2 , A3 , 3 , D3 
H , 1 , R4 , 2 , A4 , 3 , D4 
H , 2 , R5 , 1 , A5 , 3 , D5 
H , 2 , R6 , 1 , A6 , 6 , D6 
C , 2 , R7 , 1 , A7 , 6 , D7 
H , 8 , R8 , 2 , A8 , 1 , D8 
C , 8 , R9 , 2 , A9 , 9 , D9 
H , 8 , R10, 2 , A10, 9 , D10
H , 10, R11, 8 , A11, 2 , D11
H , 10, R12, 8 , A12, 12, D12
H , 10, R13, 8 , A13, 12, D13

R1  = 2.88349  
R2  = 2.09834   A2  = 109.59   
R3  = 2.09931   A3  = 110.339   D3  = 240.374        
R4  = 2.09931   A4  = 110.339   D4  = 119.626        
R5  = 2.1013    A5  = 108.825   D5  = 59.1628        
R6  = 2.1013    A6  = 108.825   D6  = 241.674        
R7  = 2.89633   A7  = 111.141   D7  = 120.837        
R8  = 2.1013    A8  = 109.643   D8  = 300.354        
R9  = 2.88349   A9  = 111.141   D9  = 239.646        
R10 = 2.1013    A10 = 109.643   D10 = 119.291        
R11 = 2.09931   A11 = 110.339   D11 = 60.3742        
R12 = 2.09931   A12 = 110.339   D12 = 239.252        
R13 = 2.09834   A13 = 109.59    D13 = 119.626        

no constants
Rotational bond dihedral angles: D5, D8, D11

Rotational groups:
D5  C3H7       C1H3      
D8  C2H5       C2H5      
D11 C1H3       C3H7      

D5    C2,H14,C10,C8,H6,H7,H9,H11,H12,H13   C1,H5,H3,H4
D8    C8,H14,C10,H9,H11,H12,H13   C2,H7,C1,H3,H4,H5,H6
D11   C10,H14,H12,H13   C8,H11,C2,C1,H3,H4,H5,H6,H7,H9

no beta-scision bonds
